src/firefly/js/api/ApiBuild.js

```diff
@@ -49,16 +49,28 @@ import {buildViewerApi} from './ApiViewer.js';
 import './ApiStyle.css';
 
 
-/**
+/*
```

Comment:
I am in favor of @tgoldina's suggestion; using @public instead of /** to show/hide functions from jsdoc.